test/unitTests/logging/OmnisharpChannelObserver.test.ts

```diff
@@ -11,14 +11,19 @@ suite("OmnisharpChannelObserver", () => {
 
     let hasShown: boolean;
     let hasCleared: boolean;
+    let preserveFocus: boolean;
     let observer: OmnisharpChannelObserver;
 
     setup(() => {
         hasShown = false;
         hasCleared = false;
+        preserveFocus = false;
         observer = new OmnisharpChannelObserver({
             ...getNullChannel(),
-            show: () => { hasShown = true; },
+            show: (preserve) => {
```

Comment:
Do we ever call this with false?

Reply:
Is there a case where we want to steal the focus

Reply:
As of now I couldnt think of any particular case hence right now we are using true everywhere

Reply:
Why not make it even easier and remove the paramter/always pass true in this method?